src/routes/graphql.ts

```diff
@@ -193,7 +194,7 @@ export const graphql = async (app: Express) => {
         defaultMaxAge: CACHE_TTL.PUBLIC_QUERY,
       }),
       responseCachePlugin({
-        redis: connections.redis,
+        redis: connections.redis as Redis,
```

Comment:
`@matters/apollo-response-cache` should support `Cluster` instance, type assertion here for temp use.